whirlpool/eventsocket.py

```diff
@@ -133,9 +148,10 @@ async def _run(self):
                             connected_msg_done = True
                             continue
 
-                        if not subscribe_msg_done:
+                        elif not subscribe_msg_done:
                             subscribe_msg_done = True
                             await self._con_up_listener()
+                            self._msg_listener(None)
```

Comment:
This is part of the "original" mistake - needing to tell HA new data is available.

Reply:
See previous comment....